docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+version: "3.3"
+services:
+  star-tides:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    restart: always
+    volumes:
+      - ./:/app
+    depends_on:
+      - mongodb_container
+    ports:
+      - 5000:5000
+  test:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    restart: always
+    volumes:
+      - ./:/app
+    depends_on:
+      - mongodb_container
+    ports:
+      - 5001:5000
```

Comment:
Does this need to be `5001:5001`? Or are we purposefully multiplexing onto the same container port?

Reply:
5001 is the host port, 5000 is in the container. Doing this has no effect because all of the tests take place within the container.

Reply:
Comments added. Pushing up soon.